org.kde.klevernotes.json

```diff
@@ -0,0 +1,61 @@
+{
+    "id": "org.kde.klevernotes",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.6",
+    "sdk": "org.kde.Sdk",
+    "base": "io.qt.qtwebengine.BaseApp",
+    "base-version": "6.6",
+    "command": "klevernotes",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=home",
+        "--env=QML_IMPORT_PATH=/app/qml",
+        "--env=QTWEBENGINE_DICTIONARIES_PATH=/app/qtwebengine_dictionaries",
+        "--talk-name=org.freedesktop.Flatpak"
```

Comment:
this needs an exception why is this needed ?

Reply:
It seems like I can remove `"--env=QTWEBENGINE_DICTIONARIES_PATH=/app/qtwebengine_dictionaries"` without any issue.

- `"--env=QML_IMPORT_PATH=/app/qml"` is for the WebEngineView, it won't load without it.
- `"--talk-name=org.freedesktop.Flatpak"` is used to be able to run external program, for example, syntax highlighters.

Reply:
I'm talking about `org.freedesktop.Flatpak`. what syntax highlighters are these and can these be added to the Flatpak package ?

Reply:
I'm not sure it could be added as a Flatpak.

It basically rely on external syntax highlighting to do the work, such as [chroma](https://github.com/alecthomas/chroma) or [pygments](https://pygments.org/).

Reply:
Those are optionnal, it is up to the user to decide if they want to enable it or not. If they want it, they just install the required package and the option will appear, if not, the app is still completely usable.

Reply:
if it's optional why have this permission by default for everyone ? Like you said those wanting it can add it themselves.

Reply:
I'm not very familiar with the way flatpak works. 
If this can be added afterward then I'm okay to remove it from this build, no problem.

I think we've covered all the issues with this initial commit, I'll update the PR based on your feedback, thank you.

Reply:
> Those are optionnal, it is up to the user to decide if they want to enable it or not. If they want it, they just install the required package and the option will appear, if not, the app is still completely usable.

Enabled vs installed.

My take is that they should be added to the flatpak so that the user can enable. Otherwise there is no point. It's not user friendly.

Reply:
I currently allow `chroma`, `pygmentize` and `ksynthaxhighlighter` for the external highlighters to choose from.
I also allow the use of `plantuml` to generate UML graph.

All of those are optionnals and disabled by default. To use them you install the required package and enable the options, that's it.

Having them installed by default would mean that I would need to ship 4 external program on top of KleverNotes inside this flatpak. 
What if I want to support more things in this way in the future ? Even more programs into the flatpak ?

I don't like this idea.

Reply:
Then don't have this permission by default, users can add it with `flatpak override --talk-name=org.freedesktop.Flatpak` or through Flatseal or I think KDE settings allow overriding permissions as well.